setup.py

```diff
@@ -94,7 +94,7 @@
         "rich>=10.14,<11",
         "tqdm>=4.62.3,<5.0",
         "typing-extensions ; python_version<'3.8'",
-        "web3[tester]==5.27.0",
+        "web3[tester]==5.26.0",
```

Comment:
I guess this doesn't work after all?

Reply:
Oh oops I don't think I meant to undo that.
I had some weird issue where I had accidentally pushed this to my `origin/main`. I must have mistakenly reset this here.